gitoid/Makefile

```diff
@@ -27,7 +31,7 @@ c_test:
 	
 	@echo "--------------------------------------------------"
 	@echo "BUILDING C TEST FILE"
-	@gcc --std=c11 -I"${INCLUDE_DIR}" -L"${LIB_DIR}" -l"${LIB_NAME}" -o"${TEST_EXE_FILE}" "${TEST_SRC_FILE}"
+	@gcc --std=c99 -I"${INCLUDE_DIR}" -L"${LIB_DIR}" -l"${LIB_NAME}" -o"${TEST_EXE_FILE}" "${TEST_SRC_FILE}"
```

Comment:
Was able to downgrade to C99 instead of C11 without issue, which is good in terms of keeping ourselves to a more accessible minimum C standard version.